cmd/containerd-shim-runhcs-v1/serve.go

```diff
@@ -221,7 +221,7 @@ var serveCommand = cli.Command{
 		case err := <-serrs:
 			return err
 		case <-time.After(2 * time.Millisecond):
-			// TODO: JTERRY75 this is terrible code. Contribue a change to
+			// TODO: JTERRY75 this is terrible code. Contribute a change to
```

Comment:
Also, I dont think I'm ever doing this work haha. Maybe @kevpar will when he gets bored some day. We can probably remove the comment

Reply:
I removed your name, but I think its a good idea nonetheless

Reply:
Thanks :)